js/layout-javascript/small-h-card-code.js

```diff
@@ -133,6 +133,9 @@ DynamicList.prototype.attachObservers = function() {
       }
 
       beforeOpen.then(function() {
+        $(event.target).parents('.small-h-card-list-wrapper').addClass('hidden');
```

Comment:
Please explain why this isn't `_this.$container.find('.small-h-card-list-wrapper')`